apps/api/e2e/utils/server.ts

```diff
@@ -1,6 +1,6 @@
 import axios from "axios";
 import { ChildProcess, spawn } from "child_process";
-import { Server } from "http";
+import kill from "tree-kill";
```

Comment:
This package is a new dependency and is key to actually killing all the processes that get started here.